cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/tests/snowbridge.rs

```diff
@@ -184,8 +184,8 @@ fn construct_extrinsic(
 		pallet_transaction_payment::ChargeTransactionPayment::<Runtime>::from(0),
 		BridgeRejectObsoleteHeadersAndMessages::default(),
 		(bridge_to_rococo_config::OnBridgeHubWestendRefundBridgeHubRococoMessages::default(),),
-		cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim::new(),
 		frame_metadata_hash_extension::CheckMetadataHash::<Runtime>::new(false),
+		cumulus_primitives_storage_weight_reclaim::StorageWeightReclaim::new(),
```

Comment:
Why was the order changed? To also include the weight of the metadata check?

Reply:
It's not too important because `StorageWeightReclaim` currently has some issues described in a bunch of other PRs. There is [an effort to fix it in this `TransactionExtension` context](https://github.com/paritytech/polkadot-sdk/pull/5234), which we will rebase and merge after this PR is merged. Then, with that change, all existing `TransactionExtension` pipelines need to be rearranged anyway.